src/helpers/highlight.ts

```diff
@@ -16,3 +16,8 @@ export async function storeMsg(id, ipfs, address, version, timestamp, space, typ
     }
   ]);
 }
+
+export async function isDuplicateMsg(id: string) {
```

Comment:
```suggestion
export async function doesMessageExist(id: string) Promise<boolean> {
```

Reply:
what did we decide on this 😄 

GPT says:
<img width="816" alt="image" src="https://github.com/snapshot-labs/snapshot-sequencer/assets/15967809/d9191c0e-5ad5-4437-a99d-fc6b679f92ab">

Reply:
I propose this name, based on the function functionality. It says whether a message with a passed `id` exists or not. `isDuplicateMsg` sounds more like: "check duplicates somewhere", but it's not duplicated, because we didn't save it.
p.s.: I don't think that we need to spend much time on this.